shell/scl_source

```diff
@@ -7,9 +7,9 @@ Don't use this script outside of SCL scriptlets!
 Options:
     -h, --help    display this help and exit"
 
-if [ $# -eq 0 -o $1 = "-h" -o $1 = "--help" ]; then
```

Comment:
The `||` is probably useless change.  You only need the double quotes.

Reply:
Brackets are to read, but this is a cosmetical issue. I will change it back to `-o` with double-quotes.